navigation/src/main/java/com/graphhopper/navigation/NavigateResponseConverter.java

```diff
@@ -174,6 +174,22 @@ private static ObjectNode putInstruction(PointList points, InstructionList instr
                 if (intersectionValue.containsKey("out")) {
                     intersection.put("out", (int) intersectionValue.get("out"));
                 }
+                // lanes
+                if (!instruction.getInstructionDetails().isEmpty()) {
+                    InstructionDetails lastDetail = instruction.getInstructionDetails().stream()
+                        .sorted(Comparator.comparingDouble(InstructionDetails::getBeforeTurn))
+                        .findFirst()
+                        .get();
```

Comment:
I chose the last lane information because in intersections we can't have more than one lane info. I checked the docs and the mapbox API and it had similar results